Gemfile

```diff
@@ -57,4 +57,8 @@ gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 
 gem 'devise'
 
+gem 'omniauth-rails_csrf_protection'
```

Comment:
why do we need this?

Reply:
sir ye authenticity error nhi aaye uske liye hai

Reply:
use these gem 

```
gem 'omniauth', '~> 1.9.1'
gem 'omniauth-google-oauth2'
```